internal/graphql/model/operator_helper.go

```diff
@@ -25,21 +25,21 @@ const (
 	MeshsyncSubject = "meshery.meshsync.core"
 	BrokerQueue     = "meshery"
 
-	operatorYaml = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/manifests/default.yaml"
-	brokerYaml   = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_broker.yaml"
+	operatorHelmChart = "meshery-operator"
+	brokerHelmChart   = "meshery-broker"
 )
 
 func Initialize(client *mesherykube.Client, delete bool) error {
-	// installOperator
-	err := applyYaml(client, delete, operatorYaml)
+	// install CRDs and operaotr
+	err := installUsingHelm(client, delete, operatorHelmChart)
```

Comment:
Here we can use the `meshery-operator` chart as the mainifest though the set all the sub-charts to false. We can set the default value `false`  to the `meshery-operator` sub-charts, and dynamic set the value though `mesheryctl` and internal function.

https://github.com/meshery/meshery/blob/d43bd16ddabb0f650434e3021324e0f712e61134/install/kubernetes/helm/meshery-operator/values.yaml#L157-L160